recipes/opentelemetry-cpp/all/conandata.yml

```
@@ -19,6 +22,10 @@ sources:
     sha256: "32f12ff15ec257e3462883f84bc291c2d5dc30055604c12ec4b46a36dfa3f189"
 
 patches:
+  "1.7.0":
+    - patch_file: "patches/1.7.0-0001-fix-cmake.patch"
+      patch_description: "fix lack of linking libraries"
+      patch_type: "backport"
```

Can you pretty please apply this to the test of them, I am trying to be strict to ensure they are a reliable source of information 🙏 

I know you did not add all of them but I would appreciate help correcting it :)

```suggestion
      patch_description: "fix lack of linking libraries due to conan not generating the variables that are expected"
      patch_type: "conan"
```

---

No worries, have updated to your suggested wording.